api/proto/device/v1alpha1/gpu.proto

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package nvidia.nvsentinel.v1alpha1;
+
+option go_package = "github.com/nvidia/nvsentinel/api/gen/go/device/v1alpha1;v1alpha1";
+
+import "google/api/annotations.proto";
+import "google/api/http.proto";
+import "google/protobuf/timestamp.proto";
+
+// ==========================================
+// Resource Definitions
+// ==========================================
+
+// Gpu represents a single GPU resource in Kubernetes.
```

Comment:
should we remove the Kubernetes wording? this API could be also used by SLURM and other similar tools